src/components/overlayCookie/overlayCookie.module.scss

```diff
@@ -4,31 +4,12 @@
 
 
 .cookieWrapper {
-  width: 100%;
-  position: fixed;
-  bottom: 0;
   background-color: $color-tertiary;
   opacity: 0.9;
   z-index: index(alert);
-
 }
 
 .cookieContent {
-  display: flex;
   @include generalAlignment();
   color: $color-grey-dark;
```

Comment:
I think you can remove all this styles and use tailwind, right?

Reply:
Yes, I am trying hahahhaha